code/src/layouts/modals/auth/AuthModal.tsx

```diff
@@ -7,43 +7,47 @@ import AppIconButton from "@/components/app-icon-button/AppIconButton";
 import AppTypography from "@/components/app-typography/AppTypography";
 import AppButton from "@/components/app-button/AppButton";
 
-import LoginForm from "@/layouts/modals/auth/components/login-form/LoginForm";
-import SignupForm from "@/layouts/modals/auth/components/signup-form/SignupForm";
+import SignInForm from "@/layouts/modals/auth/components/sign-in-form/SignInForm";
+import SignUpForm from "@/layouts/modals/auth/components/sign-up-form/SignupForm";
```

Comment:
I think forms should go to `layouts` folder. I suggest creating `forms` folder to store forms there, like we did with `modals`

Reply:
Yes, I agree. Created